CMP1903_A1_2324/Testing.cs

```diff
@@ -10,15 +10,37 @@ namespace CMP1903_A1_2324
 {
     internal class Testing
     {
-         /*
+        /*
          * This class should test the Game and the Die class.
          * Create a Game object, call the methods and compare their output to expected output.
          * Create a Die object and call its method.
          * Use debug.assert() to make the comparisons and tests.
          */
 
-         //Method
+        //Method
 
-        
-    }
+        public static void TestGame()
+        {
+            // Create a Game object
+            Game game = new Game();
+
+            game.StartGame();
+
+            int sum = game.SumOfDieValues;
+
+            // Assert that the sum is within the expected range based on the number of dice rolled (3 to 18 for three dice)
+            Debug.Assert(sum >= 3 && sum <= 18, $"Sum of die values ({sum}) out of range 3 to 18");
+        }
 
+        public static void TestDie()
+        {
+            // Create a Die object
+            Die die = new Die(new Random());
+
+            // Tests that the die roll is inbetween 1 and 6
+            int rollResult = die.Roll();
+            Debug.Assert(rollResult >= 1 && rollResult <= 6, $"Die roll result ({rollResult}) out of range.");
+
+        }
+    }
+}
```

Comment:
Mitchell Review:
Student ID 27607763
Have the requirements been met: There are the 3 required classes being die class, a die class and a testing class. 
The die class holds a value and returns a integer.
3 die are created they are rolled and the are added together to get the sum
Dice rolls cant be done continuously but that is an extra task
The testing class uses the debug.assert() to check if a single die can be bigger than 6 or smaller than 1 which a dice in real life cant do also check if the sum of all dice cant be smaller than 3 and also cant be bigger than 18 
This mean that task 1 is done

Is the code formatted using the Style Guidelines correctly?
The formats follow the way the forked code suggest so that i would say that it is formatted correctly

Is the code easy to read?
The code uses comments to describe how it works and also the variables are named sensibly so I think that it is easily read  

 Are different errors handled correctly?
Yes I mentioned this early when talking about debug.assert()
#debug.assert() to check if a single die can be bigger than 6 or smaller than 1 which a dice in real life cant do also check if the sum of all dice cant be smaller than 3 and also cant be bigger than 18#
since the user can input anything no user error can happen